src/sites/dMCDA.jl

```diff
@@ -37,6 +37,65 @@ struct DMCDA_vars  # {V, I, F, M} where V <: Vector
     wtzonesshade # ::F
 end
 
+"""
+    DMCDA_vars(domain::Domain, criteria::NamedVector, site_ids::AbstractArray, sumcover::AbstractArray, area_to_seed::Float64)::DMCDA_vars
+ 
+Constuctor for DMCDA variable type, where criteria are defined as a NamedVector.
+"""
+function DMCDA_vars(domain::Domain, criteria::NamedVector, site_ids::AbstractArray, sumcover::AbstractArray, area_to_seed::Float64)::DMCDA_vars
+    # Site Data
+    site_d = domain.site_data
+    nsites = size(site_d, 1)
+    area = site_area(domain)
+
+    mcda_vars = DMCDA_vars(
+        site_ids,
+        domain.sim_constants.nsiteint,
```

Comment:
Please mark these for later clean up

`nsiteint` -> `n_site_int` (actually, what does "int" mean in this case? It's not integer...)
`prioritysites` -> `priority_sites`

etc

Reply:
Via a comment or an issue?

Int in n_site_int stands for intervention (as in number of sites to intervene at), but definitely a bit confusing. Maybe change to n_site_intervene or something?

Reply:
Your own personal note.

Standardize to `snake_case` as you come across these please.